libsql-server/src/schema/db.rs

```diff
@@ -95,6 +95,19 @@ pub(crate) fn has_pending_migration_jobs(
     Ok(has_pending)
 }
 
+pub(crate) fn schema_has_linked_dbs(
+    conn: &rusqlite::Connection,
+    schema: &NamespaceName,
+) -> Result<bool, Error> {
+    let has_linked = conn.query_row(
+        "SELECT count(1) FROM (SELECT 0 FROM shared_schema_links WHERE shared_schema_name = ? LIMIT 1)",
```

Comment:
Why not just `SELECT count(1) FROM shared_schema_links WHERE shared_schema_name = ? LIMIT 1`?

Reply:
count always return a single row, while that would still work, it would perform a full db scan, whereas this query stops when it hits the first matching row, and returns 1 or 0